src/md5/md5.cc

```diff
@@ -119,11 +119,11 @@ void MD5::update ( const uint1 *input, uint4 input_length) {
 
 void MD5::update(FILE *file){
 
-  unsigned char buffer[1024];
-  int len;
+  unsigned char fileBuffer[1024];
```

Comment:
Any reason for the rename?

Reply:
To get rid of the shadow warnings caused by the class member with the same name. This is the reason for the various other renames as well.